src/validation.cpp

```diff
@@ -1524,10 +1524,8 @@ static uint32_t GetNextBlockScriptFlags(const Consensus::Params &params,
     // Always enforce CLEANSTACK
     flags |= SCRIPT_VERIFY_CLEANSTACK;
 
-    // If the UAHF is enabled, we start accepting replay protected txns
-    if (IsUAHFenabled(params, pindex)) {
```

Comment:
Are we going to remove these functions as some point?

Reply:
Yes. Make an issue